corpus2alpino/annotators/enrich_lassy.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+"""
+Allows enriching nodes of a Lassy annotation using a dictionary
+"""
+from typing import cast, Dict, List
+
+import csv
+from lxml import etree
+
+from corpus2alpino.abstracts import Annotator
+from corpus2alpino.annotators.alpino import ANNOTATION_KEY
+from corpus2alpino.log import LogSingleton
+from corpus2alpino.models import Document, Utterance
+
+
+class EnrichLassyAnnotator(Annotator):
+    def __init__(self, enrichment_file: str):
+        """Creates an annotator which will add additional attributes to
+        nodes when they match the properties specified in an enrichment
+        file.
+
+        Arguments:
+            enrichment_file {str} -- Path to a comma-separated enrichment file
+            matching attributes columns should be prefixed with @, assignments
+            without. For example the following CSV would match 
```

Comment:
I understand what you mean with the sentence "Matching attributes ... assignments without", but perhaps the following is clearer:

> The columns with a name starting with an `@` character determine the attributes to match against. A row in the enrichment file with a value (e.g. `noun`) in this column (e.g. `@pos`) will match `<node>`s that have `pos="noun"`. The columns with a name **not** starting with an `@` character name new attribute values for the matching `<node>`.

You can then follow up with the same example.